app/femr/util/startup/DatabaseSeeder.java

```diff
@@ -1048,4 +1058,16 @@ private void seedAdminUser() {
             userRepository.create(superUser);
         }
     }
+
+    private void seedUserRoles() {
+      // Other roles are in database from early evolutions
+      List<? extends IRole> roles = roleRepository.findAll(Role.class);
+      if (!containRole(roles, "Manager")) {
+        // Manager role doesn't exist, add it
+        Role newRole = new Role();
+        newRole.setId(Roles.MANAGER);
```

Comment:
Yeah, I added it so it'll fail noisily instead of silently. Roles.MANAGER is hard coded to 7 in Roles.java, so if some other role (with an id of 7) exists for some reason, everything using Roles.MANAGER won't work properly.

But I can remove it if you don't see a point.